src/main/markdown/release-notes.md

```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
 **Note** - M1 = first milestone, RC1 = first release candidate
 
 * * *
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
+
+- GSS is no longer considered as experimental and old /CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated.
+
+### Compiler changes
+
+- Java 8 syntax is now supported by the compiler.
+- New optimizer framework that reruns the optimization process only on the part of the AST that were impacted by the changes since its last run. That speedup compilation of large application.
+-
+- Many bugfixes.
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
```

Comment:
Started using ES6 Maps when available for HashMap/HashSet that yields up 3x performance improvements.